astro/src/components/quickstarts/quickstart-ngrok.mdx

```diff
@@ -0,0 +1,16 @@
+Your FusionAuth instance is now running on a different machine (your computer) than the mobile app will run (either a real device or an emulator), which means that it won't be able to access `localhost`.
```

Comment:
I'm not crazy about the ()'s and I had to read this a few times before it made sense. I can't really think of a better way to put it other than making it a little more wordy.  I.E.

Your FusionAuth instance in now running on you machine. The mobile app will run on a real device or an emulator. This means the mobile app will not be able to access the FusionAuth instance using the `localhost` address.

This may be a preference issue so this is just a suggestion.